programs/airdrop/src/instructions/claim.rs

```diff
@@ -6,10 +6,10 @@ use crate::State;
 use crate::VAULT_SEED;
 
 #[derive(Accounts)]
-#[instruction(amount: u64, verifier_data: Vec<u8>)]
+#[instruction(amount: u64)]
 pub struct Claim<'info> {
-    /// Mutable because might need to pay rent in verifier.
-    #[account(mut)]
+    // TODO: Add one time hack for backwards compatibility of DUAL CSA
```

Comment:
Do you mean remove? We need backwards compatibility for CSA now right?

Reply:
The hack isnt in there yet. will need to bypass for our CSA